src/Symfony/Component/Security/Http/Authenticator/AbstractAuthenticator.php

```diff
@@ -28,8 +28,6 @@ abstract class AbstractAuthenticator implements AuthenticatorInterface
     /**
      * Shortcut to create a PostAuthenticationToken for you, if you don't really
      * care about which authenticated token you're using.
-     *
-     * @return PostAuthenticationToken
```

Comment:
`AbstractAuthenticator` cannot force child classes to return an instance of `PostAuthenticationToken`

`FormLoginAuthenticator` doesn't btw.